output/output.go

```diff
@@ -299,6 +300,9 @@ func printTable(cmd *cobra.Command, t *Table) {
 		tw.SetHeader(t.Headers)
 	}
 	tw.AppendBulk(t.Lines)
+	for _, tup := range t.ColumnMinWidths {
+		tw.SetColMinWidth(tup[0], tup[1])
```

Comment:
@linkous8 , two questions:
1. Wouldn't it be necessary to check if t.ColumnMinWidths is non-nil here?
2. Given the number of table features we had to expose to support the event tail/list, are we stretching the limits of what the common table handler can provide and, instead, use custom table formatting for the events (or even create a separate output helper for this data type). I know this is somewhat subjective but I am suggesting to be considered.

Reply:
1. Using range against a nil-slice produces zero iterations without panicking. I verified this via pen-testing and also confirmed its part of the spec:

	> For a nil slice, the number of iterations is 0.

	* https://github.com/golang/go/issues/1713
	* https://go.dev/ref/spec#For_range
	* https://groups.google.com/g/golang-nuts/c/gVgVLQU1FFE
2. All features added to the Table struct are not specific to events and may be of use to others implementing features using table output.

Reply:
Agree. Thanks for the links